src/main/java/racingcar/service/RacingGameService.java

```diff
@@ -0,0 +1,62 @@
+package racingcar.service;
+
+import racingcar.domain.Cars;
+import racingcar.domain.car.Car;
+import racingcar.domain.car.MovingStrategy;
+import racingcar.dto.RacingCarRequestDto;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class RacingGameService {
+    private static final String DELIMITER = ",";
+    private static final int INT_ZERO = 0;
+    private static final int INT_ONE = 1;
+
+    public List<Cars> executeRacingGameNumberOfTimes(RacingCarRequestDto racingCarRequestDto, MovingStrategy movingStrategy) {
+        String[] racingCarNames = racingCarRequestDto.getRacingCarNames().split(DELIMITER);
+        List<Car> cars = createCars(racingCarNames);
+        List<Cars> racingScores = new ArrayList<>();
+
+        for (int i = 0; i < racingCarRequestDto.getNumberOfTime(); i++) {
+            racingScores.add(executeRacingGame(cars, movingStrategy));
+        }
+
+        return racingScores;
+    }
+
+    private List<Car> createCars(String[] names) {
```

Comment:
일급 컬렉션인 `Cars`를 사용해 보세요.

```suggestion
    private Cars createCars(String[] names) {
```